src/index.json

```diff
@@ -1188,6 +1188,38 @@
                     "version": "0.4.55"
                 },
                 "sha256Digest": "d6fa0ec6dfffa82941d9b93415b317f5aee42191c3f6193fbc5842f36822cc48"
+            },
+            {
+                "downloadUrl": "https://azurecliaks.blob.core.windows.net/azure-cli-extension/aks_preview-0.4.56-py2.py3-none-any.whl",
```

Comment:
I just found in latest version aks-preview, the wheel file is stored in storage account `azurecliprod`, how can I upload the extension to it?

Reply:
Just realized the blob and index will be updated by CI, reverted the index change.